### ISSUE TITLE
Pybel not compatible with unicode_literals

### ISSUE THREAD

**Issue by mwojcikowski** [openbabel/openbabel#325]
In Python 2.7 there is a bug when unicode_leterals are loaded

```
from __future__ import unicode_leterals
import pybel
m = pybel.readstring('smi', 'c1ccccc1')
```

```
TypeError                                 Traceback (most recent call last)
<ipython-input-3-c6b6881f6c6d> in <module>()
----> 1 m = pybel.readstring('smi', 'c1ccccc1')

/home/maciek/miniconda/lib/python2.7/site-packages/pybel.pyc in readstring(format, string, opt)
    195     obconversion = ob.OBConversion()
    196 
--> 197     formatok = obconversion.SetInFormat(format)
    198     if not formatok:
    199         raise ValueError("%s is not a recognised Open Babel format" % format)

TypeError: in method 'OBConversion_SetInFormat', argument 2 of type 'char const *'
```
Posible [solution](http://swig-user.narkive.com/pTuPgRC9/typemap-of-python-unicode-to-c-std-string-in-utf-8-encoding) (?):

**Comment by baoilleach** [comment]
I don't really regard this as a bug. Support for unicode is not a feature.

**Comment by mwojcikowski** [comment]
You can't write forward compatible code, which is a pain... Unless that's also a feature ;) Should we add this feature then? I think there is no downside to that, beside probably being a ns slower.

**Comment by baoilleach** [comment]
If you want to. Are you saying that this also a problem in Python 3?

**Comment by mwojcikowski** [comment]
No, Python 3.0 is fine. I think SWIG guess that in Python 2.x byte -> char and in Python 3.x unicode -> char.

**Comment by mwojcikowski** [comment]
Fixed in #326
